pkg/util/node.go

```diff
@@ -12,3 +22,74 @@ func IsNodeReady(node *v1.Node) bool {
 	}
 	return false
 }
+
+// ThisNode identifies the node object that is representing the system ThisNode is execited on.
+func ThisNode(nodes []v1.Node) (*v1.Node, error) {
+
+	//1. try to match by machineID
+	machineID, err := GetMachineID()
+	//blacklist machine-ids baked into v1.7.5_coreos.0, v1.8.5_coreos.0 and v1.9.0_coreos.0 hyperkube images
+	for _, blacklisted := range []string{"833e0926ee21aed71ec075d726cbcfe0", "40beb5eb909e171860ceee669da56e1d"} {
```

Comment:
Do we need to update this for each new image? I started to build our own images with 1.10 since CoreOS is so slow with publishing their builds. It also contains 300MB Calico which we don't need (yet). Maybe I can just delete the baked in ID in the build?